man/systemd-journal-gatewayd.service.xml

```diff
@@ -277,9 +277,13 @@
     <para>
       <option>Range: entries=<replaceable>cursor</replaceable>[[:<replaceable>num_skip</replaceable>]:<replaceable>num_entries</replaceable>]</option>
     </para>
+    <para>
+      <option>Range: time=[<replaceable>since</replaceable>]:[<replaceable>until</replaceable>][[:<replaceable>num_skip</replaceable>]:<replaceable>num_entries</replaceable>]</option>
```

Comment:
maybe name this realtime, to underline that this is not CLOCK_MONOTONIC or so

Reply:
OK!